internal/controllers/rbac.go

```diff
@@ -292,7 +292,7 @@ func (r *CryostatReconciler) createOrUpdateClusterRoleBinding(ctx context.Contex
 	if err != nil {
 		return err
 	}
-	r.Log.Info(fmt.Sprintf("Cluster Role Binding %s", op), "name", binding.Name, "namespace", binding.Namespace)
+	r.Log.Info(fmt.Sprintf("Cluster Role Binding %s", op), "name", binding.Name)
```

Comment:
Just a heads up, this is an extra bit of cleanup since Cluster Role Bindings do not have a namespace.